tests/foreman/api/test_registration.py

```diff
@@ -107,12 +107,22 @@ def test_positive_allow_reregistration_when_dmi_uuid_changed(
     """
     uuid_1 = str(uuid.uuid1())
     uuid_2 = str(uuid.uuid4())
-    rhel_contenthost.install_katello_ca(target_sat)
     target_sat.execute(f'echo \'{{"dmi.system.uuid": "{uuid_1}"}}\' > /etc/rhsm/facts/uuid.facts')
-    result = rhel_contenthost.register_contenthost(module_org.label, lce=ENVIRONMENT)
-    assert result.status == 0
-    result = rhel_contenthost.execute('subscription-manager clean')
```

Comment:
Please do not change the core step of the test!
I recommend reading thoroughly the BZ associated with test cases that you are changing to ensure you are not destroying the test logic.
See https://bugzilla.redhat.com/show_bug.cgi?id=1747177#c19

Please change `unregister` back to `clean`

Reply:
Thanks for pointing this. I missed that comment on the BZ. Changed it to `subscription-manager clean`.